config/application.rb

```diff
@@ -18,6 +18,20 @@
 
 module BattleshipWeb
   class Application < Rails::Application
+    config.action_mailer.delivery_method = :smtp
+
+    config.action_mailer.smtp_settings = {
+      address:              'smtp.sendgrid.net',
+      port:                 '587',
+      domain:               'immense-woodland-83336.herokuapp.com',
+      user_name:            ENV["SENDGRID_USERNAME"],
+      password:             ENV["SENDGRID_PASSWORD"],
+      authentication:       'plain',
+      enable_starttls_auto: true
+    }
+    # Do not swallow errors in after_commit/after_rollback callbacks.
+    # config.active_record.raise_in_transactional_callbacks = true
```

Comment:
@jmejia, @icorson3 Line 33 was included in the lesson on email, but it was throwing an error that was something like " undefined method raise_in_transactional_callbacks=" everywhere -- is it possible for you to say whether we'll need this config line?

Reply:
@ssciolist I know we already chatted about this, but based on this error I think this is probably something that has been removed. Either way, definitely don't need it.